docs/source/markdown/podman-systemd.unit.5.md

```diff
@@ -238,9 +238,14 @@ that limit the output to only the units you are debugging.
 
 ### Implicit network dependencies
 
-In the case of Container, Image and Build units, Quadlet will add dependencies on the `network-online.target`
-by adding `After=` and `Wants=` properties to the unit. This is to ensure that the network is reachable if
-an image needs to be pulled.
+In the case of Container, Image and Build units, Quadlet will add dependencies on the `network-online.target` (as root)
```

Comment:
Since we're already touching this. Why not add this to all units, or at least to `.kube`?

Reply:
I haven't given this a second thought. But yes as kube units can pull and start pasta so we definitely need to have it there.
Not sure if we need this for the other unit types though but I guess for consistency it might be easiest to add it to all units which makes documentation easier all less surprises for users

Reply:
Yes, I don't see the harm in having them, so I think it's better to add it in all types

Reply:
Good catch, Ygal!

Reply:
done for all units

Reply:
Great, thanks